index.js

```diff
@@ -0,0 +1,5 @@
+var btn = document.getElementsByClassName('btn')[0]
```

Comment:
Use latest ecmascript features

Reply:
When we do native development we should use id as selector as it should appear only once on the page